src/OpenTelemetry.Exporter.OpenTelemetryProtocol/CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 * Added configuration options for `MetricReaderType` to allow for configuring
   the `OtlpMetricExporter` to export either manually or periodically.
   ([#2674](https://github.com/open-telemetry/opentelemetry-dotnet/pull/2674))
+* Added logs on collector URI in case of connection failure in all OTLP export
```

Comment:
changelog is not really required.

Alternate suggestion:
The internal log message used when OTLP export client connection failure occurs, will now include the endpoint uri as well.

Reply:
Done